### PR TITLE
yarn install if yarn.lock missing

### PR DESCRIPTION
**Pull request by RasPhilCo**
Closes https://github.com/heroku/cli-engine/issues/78

### CODE REVIEW

File: src/plugins.js [hunk 1]
```diff
@@ -95,6 +98,11 @@ export default class Plugins {
 
   async update () {
     if (this.userPlugins.list().length === 0) return
+    let client = path.join(this.config.dataDir, 'client')
+    if (!fs.existsSync(path.join(client, 'yarn.lock'))) {
+      let yarn = new Yarn(this.out)
+      await yarn.exec([], {cwd: client})
```
Comment (RasPhilCo) [inline src/plugins.js]: might not need the `cwd` here, think `client` is the cwd

Comment (jdx) [reply src/plugins.js]: I think you do, otherwise it will run from wherever the CLI was executed right?

Comment (RasPhilCo) [reply src/plugins.js]: That's what I thought, but other uses of `exec` don't. Will investigate.

Comment (jdx) [reply src/plugins.js]: nevermind, you don't need to, it will default to that directory in yarn.js

**Review by jdx — APPROVED** [review]
easy mode

**Pull request merged**